src/res.rc

```diff
@@ -38,6 +38,7 @@ BEGIN
     VK_ESCAPE,  IDM_ESCAPE,     NOINVERT, VIRTKEY
     VK_RETURN,  IDM_OPEN,       NOINVERT, VIRTKEY
     VK_RETURN,  IDM_OPEN,       NOINVERT, VIRTKEY, SHIFT
+    VK_RETURN,  IDM_OPEN,       NOINVERT, VIRTKEY, CONTROL

```

Comment:
Can I ask why we do this? I know this PR is just following a pattern, but it looks like this causes User32 to match the Control+Return combination, then map it to the same ID as other keystrokes, then Winfile peeks at whether the user is currently holding down Control to determine which action to take, rather than having a unique ID that makes this unambiguous.

Winfile is checking the state of Control when processing the message, not when the message is sent, so it becomes possible to do things like Press Enter, Release Enter, Press Control, and have Winfile assume it's Control+Enter.